python/tests/test_cli/test_apps.py

```diff
@@ -2,43 +2,79 @@
 Test `composio apps` command group.
 """
 
+import random
+import typing as t
 from pathlib import Path
 
+import pytest
+
 from composio.client import enums
 
 from tests.test_cli.base import BaseCliTest
 
 
+ENUM_CLS = (
+    enums.App,
+    enums.Action,
+    enums.Tag,
+    enums.Trigger,
+)
+
+
 class TestList(BaseCliTest):
     """Test apps list."""
 
     def test_list(self) -> None:
         """Test list all apps."""
-        result = self.run("apps")
-        assert result.exit_code == 0, result.stderr
-        assert "Showing all apps" in result.stdout
-        assert "github" in result.stdout
+        self.run("apps")
+        self.assert_exit_code(code=0)
+        self.assert_stdout(match="Showing all apps")
+        self.assert_stdout(match="github")
 
 
 class TestUpdate(BaseCliTest):
     """Test apps update."""
 
-    file: Path
-    content: str
+    files: t.Dict[Path, str]
 
     def setup_method(self) -> None:
         """Setup update test."""
-        self.file = Path(enums.__file__)
-        self.content = self.file.read_text(encoding="utf-8")
+        self.files = {}
+        for file in Path(enums.__file__).parent.iterdir():
+            if file.is_dir():
+                continue
+            self.files[file] = file.read_text(encoding="utf-8")
 
     def teardown_method(self) -> None:
         """Restore original enums file after testing."""
-        self.file.write_text(data=self.content, encoding="utf-8")
+        for file, content in self.files.items():
+            file.write_text(data=content, encoding="utf-8")
 
+    def test_update_not_required(self) -> None:
+        """Test app enums update."""
+        self.run("apps", "update")
+        self.assert_exit_code(code=0)
+        for cls in ENUM_CLS:
+            self.assert_stdout(f"⚠️ {cls.__name__}s does not require update")
+
+    @pytest.mark.skip(
+        reason="Needs investigation, this test fails in CI",
+    )
     def test_update(self) -> None:
```

Comment:
The `test_update` method in `TestUpdate` class is currently skipped with the reason "Needs investigation, this test fails in CI". It's important to investigate and fix this failing test before merging the PR. Skipping tests, especially those that verify critical functionality like enum updates, can lead to potential issues in production. Please investigate the cause of the failure in CI and either fix the test or provide a detailed explanation if the test is no longer relevant.